### PR TITLE
064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #932

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+-- [GeneralResultCodes] INVALID_DATA wrong json syntax
+-- [MOBILE_API] SetAudioStreamingIndicator
+-- [HMI_API] [MOBILE_API] AudioStreamingIndicator enum
+--
+-- Description:
+-- In case the request comes to SDL with wrong json syntax, SDL must respond with
+-- resultCode "INVALID_DATA" and success:"false" value.
+--
+-- 1. Used preconditions
+-- Do not allow SetAudioStreamingIndicator RPC by policy
```
Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua]: @istoimenova Please add directly disallowing SetAudioStreamingIndicator from script

Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua]: @dboltovskyi, please check update in bdf8eb4567844300d2a4f19b903df066cbad3d54

File: test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	}
+
+  self.mobileSession:Send(msg)
+
+  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", {}):Times(0)
+  
+  EXPECT_RESPONSE(self.mobileSession.correlationId, {success = false, resultCode = "DISALLOWED"})
+  EXPECT_NOTIFICATION("OnHashChange",{}):Times(0)
+end
+
```
Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua]: @istoimenova Plese restore preloaded file

Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua]: @OVikhrov, done in e6e77484aa3a8a7d60671f061668062ca94004c4

**Pull request merged**